styles/style.css

```diff
@@ -0,0 +1,1124 @@
+body {
+  font-family: "Montserrat", sans-serif;
+  margin: 0;
+  padding: 0;
+  color: #555555;
+}
+
+.header {
+  background-image: url("../images/baner.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 22px 190px 140px;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  color: #ffffff;
+}
+
+.header .header-nav {
+  margin-bottom: 182px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+}
+
+.header .header-nav .logo {
+  height: 20px;
+  z-index: 99;
+}
+
+.header .header-nav .nav-list {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  width: 500px;
+}
+
```

Comment:
![image](https://user-images.githubusercontent.com/44294945/69324440-23291280-0c51-11ea-95ec-98d5a9f9d698.png)
this is must be a link